purchase_order_weight_volume/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Purchase Order Weight and Volume",
+    "summary": """Display purchase order weight and volume""",
+    "version": "16.0.1.0.0",
+    "author": "Ilyas, Ooops404, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "category": "Purchase Management",
+    "license": "AGPL-3",
```

Comment:
I see no license and no (c) in all the files... can we fix that?
Maybe to be done on 15 first and ported here. 
CC @ilyasProgrammer

Reply:
@simahawk https://github.com/OCA/purchase-workflow/pull/1978